osu.Game.Rulesets.Osu/Tests/TestCaseHitCircleHidden.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using osu.Game.Rulesets.Osu.Mods;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Rulesets.Osu.Objects.Drawables;
+
+namespace osu.Game.Rulesets.Osu.Tests
+{
+    [Ignore("getting CI working")]
+    public class TestCaseHitCircleHidden : TestCaseHitCircle
+    {
+        public override IReadOnlyList<Type> RequiredTypes => new[]

```

Comment:
Move all but the mod required types to the base `TestCase` and concatenate with hidden for these derived classes (should be done for all three):
`base.RequiredTypes.Concat(new[] { typeof(OsuModHidden) }).ToList();`